x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/build_alert.ts

```diff
@@ -178,7 +178,7 @@ export const buildAlert = (
 
   const originalTime = getValidDateFromDoc({
     doc: docs[0],
-    timestampOverride: undefined,
+    primaryTimestamp: TIMESTAMP,
```

Comment:
Just commenting so I understand correctly, but this doesn't result in a functional change, and is just essentially hoisting the defaulting to `@timestamp` from inside `getValidDateFromDoc` to whatever the callee provides?

Looking at the other usages it seems everything keeps the same behavior, but just want to verify.

Reply:
Yes, that is correct. There is no functional change in this case.